docroot/modules/custom/mayflower/src/Prepare/Organisms.php

```diff
@@ -198,13 +199,19 @@ public static function prepareContactList($entity, array $options = [], array &$
     foreach ($ref_contacts as $contact) {
       // Get entity cache tags.
       $cache_tags = array_merge($cache_tags, $contact->getCacheTags());
-      if ($contact->moderation_state) {
-        if ($contact->moderation_state[0]) {
-          if ($contact->moderation_state[0]->value === MassModeration::PUBLISHED) {
-            $contacts[] = Molecules::prepareContactUs($contact, $options['groups']);
+      if ($contact instanceof NodeInterface) {
+        if ($contact->moderation_state) {
+          if ($contact->moderation_state[0]) {
+            if ($contact->moderation_state[0]->value === MassModeration::PUBLISHED) {
+              $contacts[] = Molecules::prepareContactUs($contact, $options['groups']);
+            }
           }
         }
       }
+      else {
+        $contacts[] = Molecules::prepareContactUs($contact, $options['groups']);
+      }
```

Comment:
```suggestion
      $contacts[] = Molecules::prepareContactUs($contact, $options['groups']);
```
@arthurbaghdas It looks like we should be able to remove the conditionals around this.

Reply:
<img width="549" alt="Screenshot 2024-06-27 at 09 35 58" src="https://github.com/massgov/openmass/assets/62405127/bfebd859-d7cb-4ef3-a0fa-8c92963ab232">
No that will actually cause a duplicate content. The else is for the cases when the referenced entity is a paragraph and doesnt have a moderation state.